pixel/patterns/post-list.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Title: Posts list
+ * Slug: pixel/posts-list
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:query {"query":{"perPage":"10","pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"tagName":"main","displayLayout":{"type":"list","columns":3},"layout":{"type":"constrained"}} -->
+<main class="wp-block-query"><!-- wp:post-template -->
+<!-- wp:group {"layout":{"type":"constrained"},"style":{"border":{"width":"2px"},"spacing":{"padding":{"top":"33px","right":"33px","bottom":"33px","left":"33px"}}},"borderColor":"primary","className":"pixel-shadow"} -->
+<div class="wp-block-group pixel-shadow has-border-color has-primary-border-color" style="border-width:2px;padding-top:33px;padding-right:33px;padding-bottom:33px;padding-left:33px"><!-- wp:post-date {"style":{"typography":{"textTransform":"uppercase","fontSize":"12px"}}} /-->
+
+<!-- wp:post-title {"isLink":true} /-->
+
+<!-- wp:post-excerpt /-->
+
+<!-- wp:template-part {"slug":"post-meta","theme":"themes/pixel"} /-->
+
+<!-- wp:spacer {"height":"40px"} -->
+<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:group -->
```

Comment:
Move the spacer after the group block and set it to `2rem`.

| Current | Suggested |
| --- | --- |
| <img width="258" alt="image" src="https://user-images.githubusercontent.com/1935113/188555028-b5f5bd1d-99f8-4eaa-93a0-9afd2ab0c5d0.png"> | <img width="258" alt="image" src="https://user-images.githubusercontent.com/1935113/188554700-c1a7e6b4-7020-4319-ac09-f91da82d56de.png"> |

Reply:
Done in https://github.com/Automattic/themes/pull/6481/commits/c693bfb3272693ab55581e6c5ba950ec51a85148.